proxy-node-gateway/src/main/java/uk/gov/ida/notification/session/GatewaySessionData.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.ida.notification.session;
+
+
+import org.hibernate.validator.constraints.NotEmpty;
+
+public class GatewaySessionData {
```

Comment:
Do we need `Gateway` in the name?

Reply:
Yeah, the session object itself already has a data object of type `SessionData`. This was to be clear about what's what.

Reply:
👍